src/server/editorServices.ts

```diff
@@ -447,6 +459,26 @@ namespace ts.server {
             });
         }
 
+        private loadTypesMap() {
+            if (!this.host.fileExists(this.typesMapLocation)) {
+                this.logger.info(`Provided types map file "${this.typesMapLocation}" doesn't exist`);
+                return;
+            }
+            try {
+                const raw: TypesMapFile = JSON.parse(this.host.readFile(this.typesMapLocation));

```

Comment:
Would it be worth using `ts.parseJsonText` and `ts.convertToObject` instead of `JSON.parse` so that we can get accurate line/column information for invalid JSON in the logs?